### ISSUE TITLE
Post 2.0.0 release after #10 merged?

### ISSUE THREAD

**Issue by efung** [ashdavies/rx-tasks#11]
The current README has been updated to reflect the extension methods on `Task`, e.g. `toSingle()` but there's no public release including it. Any plans to do so in the near future?

**Comment by ashdavies** [comment]
Yeah I was migrating to use the ShipKit plugin for releasing, but I was having some issue with configuration, it's coming soon.

**Comment by ashdavies** [comment]
Actually, I'm going to leave this issue open until the release has been made

**Comment by tbsandee** [comment]
seems like the release was made, yes? can this be closed?

**Comment by ashdavies** [comment]
Yep, some further documentation might be helpful, I'll create a separate issue for that

**Comment by efung** [comment]
Wait, what release was made? The latest release is 2.0.0, which doesn't include the improvements in #10.

**Comment by tbsandee** [comment]
My mistake!  I assumed that master held the current release.

**Comment by efung** [comment]
@ashdavies Can you re-open this, until a new release is created?

**Comment by ashdavies** [comment]
My intention is to resolve the new deployment plugin as a priority and address the current outstanding PR's for the next release, but I have a pretty heavy workload at the moment.

**Comment by ashdavies** [comment]
`2.1.2` is now available via [Maven Central](https://search.maven.org/artifact/io.ashdavies.rx.rxtasks/rx-tasks/2.1.2/aar)
